src/main/scala/org/scalameter/Executor.scala

```diff
@@ -1,18 +1,21 @@
 package org.scalameter
 
 
-import collection._
-import compat._
-import utils.{withGCNotification, Tree}
+import utils.Tree
 
 
 
 trait Executor {
 
+  def measurer: Measurer
+
   def run[T](setuptree: Tree[Setup[T]], reporter: Reporter, persistor: Persistor): Tree[CurveData] = {
+    val context = measurer.prepareContext(setuptree.context)
```

Comment:
The classpath entry is just one additional key.
I'd say that it is not very dangerous. Let's leave it in for now.

On 7/9/2015 1:42 AM, Krzysztof Janosz wrote:
>>    def run[T](setuptree: Tree[Setup[T]], reporter: Reporter, persistor: Persistor): Tree[CurveData] = {
>> +    val context = measurer.prepareContext(setuptree.context)
> 
> One thing has come to my mind, also the `classpath` key will be changed - it will contain the instrumented jar. 
> So the question is, should we clean the classpath (so the `cleanContext` method  should stay) or we don't care about this additional classpath entry?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/scalameter/scalameter/pull/135/files#r34211064
> 

-- 
Aleksandar Prokopec